SF iOS/SF iOS/Feed/EventsFeedViewController.m

```diff
@@ -107,7 +161,13 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    return self.dataSource.numberOfEvents;
+    NSUInteger eventCount = self.dataSource.numberOfEvents;
+    if (eventCount < 1) {
```

Comment:
`numberOfRowsInSection` feels like a weird place for any logic to live, unless we're going to return a separate `UITableViewCell` to represent no data instead of showing/hiding a view

Reply:
This also shows the "No events" view when launching the app for the first time.

Reply:
The overall first launch experience is much better but I still see a bit of glitch on first load. Do you think it makes sense to maybe hide the search bar if there is no content showing?